src/main/java/com/carrotcreative/recyclercore/inject/InjectController.java

```diff
@@ -0,0 +1,18 @@
+package com.carrotcreative.recyclercore.inject;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Created by kaushiksaurabh on 10/1/15.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface InjectController
```

Comment:
Yeah I did think this was an interesting name for this....

Let me sit on it for a little bit and I'll let you know if I come up with anything.

I'm not necessarily sold on controller, because this has both the controller and the view.